Source/WebKit/Platform/IPC/ArgumentCoders.h

```diff
@@ -491,6 +516,14 @@ template<typename KeyArg, typename MappedArg, typename HashArg, typename KeyTrai
             encoder << *it;
     }
 
+    template<typename Encoder>
+    static void encode(Encoder& encoder, HashMapType&& hashMap)
+    {
+        encoder << static_cast<uint32_t>(hashMap.size());
+        for (typename HashMapType::iterator it = hashMap.begin(), end = hashMap.end(); it != end; ++it)
```

Comment:
Can we use c++11-style iteration instead?

Reply:
Yes, of course.